src/components/QuizGameApi/index.jsx

```diff
@@ -0,0 +1,14 @@
+// we fetch those questions from opentdb website. we can pick the type of questions,the amount and the dificulty.
+// here we have a 10 questions as an amount, nature as a category ,easy, middle and hard for the difficulty and multiple choices as a type.
+// iam fetching data from a website called opentdb.com this website will give us a simple questions about the nature.
+// this is an example of the game https://lovattspuzzles.com/online-puzzles-competitions/ultimate-online-trivia-quiz.
+import shuffle from "../../components/memorygame/shuffle.jsx";
+const NATURE_CATEGORY = 17;
```

Comment:
newline above and below please

Reply:
PTAL

Reply:
i meant above NATURE_CATEGORY. there should be a space between the imports and the beginning of your code (look at other examples in the project and you'll see that everyone should have put a space between the imports and the non-import parts of the code)

Reply:
PTAL